lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +19,13 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    text1 = text.lower()
+    for punctuation1 in string.punctuation:
+        if punctuation1 in text1:
```

Comment:
why do we need to check if the symbol is in text1?

Reply:
yes, we don't
fix